Sources/RichTextKit/RichTextView_AppKit.swift

```diff
@@ -21,6 +21,8 @@ import AppKit
 open class RichTextView: NSTextView, RichTextViewComponent {
 
     // MARK: - Properties
+
+    var heightConstraint: NSLayoutConstraint?
```

Comment:
This is not used anywhere.

Reply:
yes, maybe I can convince you to make it public :)